qiskit/__init__.py

```diff
@@ -58,10 +58,11 @@
 try:
     from qiskit.providers.ibmq import IBMQ
 except ImportError:
-    warnings.warn('Could not import the IBMQ provider from the '
-                  'qiskit-ibmq-provider package. Install qiskit-ibmq-provider '
-                  'or check your installation.',
-                  RuntimeWarning)
+    pass
+#    warnings.warn('Could not import the IBMQ provider from the '
+#                  'qiskit-ibmq-provider package. Install qiskit-ibmq-provider '
+#                  'or check your installation.',
+#                  RuntimeWarning)
```

Comment:
is this left over from debugging?

Reply:
Oops yep, a stray `git commit -a` from habit. I'll fix this and the other one (which was in the conf.py)